core/frontend/src/tile/TileAdmin.ts

```diff
@@ -1274,6 +1277,13 @@ export namespace TileAdmin { // eslint-disable-line no-redeclare
      * @alpha This was primarily introduced because the electron version of certa does not serve local assets, so the tests can't locate the worker script.
      */
     decodeImdlInWorker?: boolean;
+
+    /** If true, an indexedDB database will be added to the itwinjs-batched-models tile request channel for use as a local cache.
+     * Tiles requested through this channel will first be searched for in the database, and if not found, will be requested from the backend, then stored for later requests.
+     * Default value: false
+     * @alpha
+     */
+    useIndexedDBCache?: boolean;
```

Comment:
Can you move this to `FrontendTilesOptions`?

Reply:
Moved it to `FrontendTilesOptions` and added it wherever necessary within the rest of the options functionality. Let me know if I missed something or incorrectly configured some of it.